google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -45,6 +45,8 @@ appengine.appyaml.location.label=app.yaml path\:
 appengine.dockerfile.location.browse.button=Browse for Your Dockerfile Location
 appengine.deployment.error.with.code=Deployment failed with exit code: {0}\nPlease make sure that you are using the latest version of the Google Cloud SDK.\nRun ''gcloud components update'' to update the SDK. (See: https://cloud.google.com/sdk/gcloud/reference/components/update.)
 appengine.deployment.error.during.execution=Deployment failed due to an unexpected error during the deployment execution phase.
+appengine.stopapp.error.during.execution=Failed to stop application due to an unexpected error during the execution phase.
```

Comment:
+1

On Fri, Apr 1, 2016 at 2:04 PM, Etan Shaul <notifications@github.com> wrote:

> In
> google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/571#discussion_r58243714>
> :
>
> > @@ -45,6 +45,8 @@ appengine.appyaml.location.label=app.yaml path\:
> >  appengine.dockerfile.location.browse.button=Browse for Your Dockerfile Location
> >  appengine.deployment.error.with.code=Deployment failed with exit code: {0}\nPlease make sure that you are using the latest version of the Google Cloud SDK.\nRun ''gcloud components update'' to update the SDK. (See: https://cloud.google.com/sdk/gcloud/reference/components/update.)
> >  appengine.deployment.error.during.execution=Deployment failed due to an unexpected error during the deployment execution phase.
> > +appengine.stopapp.error.during.execution=Failed to stop application due to an unexpected error during the execution phase.
>
> How about: appengine.stop.modules.version.execution.error
>
> All I changed from your suggestion is making module plural. It reads a
> little awkwardly but it makes it consistent with the underlying gcloud
> command. The stop could potentially act on multiple modules for a
> multimodule deployment.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/571/files/f30847bf0cdb604ecbec0439a415f0fd6c2328d1#r58243714>
>